spacy/default_config.cfg

```diff
@@ -6,7 +6,7 @@ init_tok2vec = null
 
 [system]
 seed = 0
-use_pytorch_for_gpu_memory = false
+gpu_allocator = ""
```

Comment:
Do we want the default to be `""` here? I know it's technically less consistent but `null` would just feel more natural?